Sources/TuistGenerator/Generator/LinkGenerator.swift

```diff
@@ -398,6 +398,7 @@ final class LinkGenerator: LinkGenerating { // swiftlint:disable:this type_body_
             condition: PlatformCondition?,
             status: LinkingStatus = .required
         ) throws {
+            guard status != .none else { return }
```

Comment:
I wonder if this change will have cascading effects due to assumptions in the [GraphTraverser](https://github.com/tuist/tuist/blob/main/Sources/TuistCore/Graph/GraphTraverser.swift) method.

Reply:
I didn't see anything affected. I compared before and after for the project trying to integrate EarlGrey and the only difference was the linking. Of course, I don't know if an issue in graph traverser would affect something other than project generation.